lib/taskana-core-test/src/test/java/acceptance/taskcomment/delete/DeleteTaskCommentAccTest.java

```diff
@@ -83,7 +83,7 @@ void should_DeleteTaskComment_For_TaskCommentId() throws Exception {
     taskService.deleteTaskComment(comment1.getId());
 
     List<TaskComment> taskCommentsAfterDeletion = taskService.getTaskComments(task1.getId());
-    assertThat(taskCommentsAfterDeletion).hasSize(0);
```

Comment:
Why do you change this? The task comment has nothing to do with the DELETED history Event